recipes/mm2plus/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.0" %}
+
+package:
+  name: mm2plus
+  version: {{ version }}
+
+source:
+  url: "https://github.com/at-cg/mm2-plus/releases/download/v{{ version }}/mm2-plus-{{ version }}_x64-linux.tar.bz2"
+  sha256: "e5b4fc2e12a475bdb46bfdbdfc563d2bae02a908f8e6f4a2c55d759e84d240de"
+
+build:
+  number: 0
+  skip: True  # [not linux]
+  run_exports:
+    - mm2plus
+
+requirements:
+  build:
+    - make
+    - {{ compiler('cxx') }}
```

Comment:
Are those needed ?
It seems the source tarball provides prebuilt binaries.

Reply:
Removed

Reply:
@gsc74  Please check the latest state of the file and test the produced artefacts!
If everything is OK we will merge the PR!
Thank you!

Reply:
@martin-g I have tested the artifacts, everything is OK.